storage-bigtable/build-proto/src/main.rs

```diff
@@ -1,4 +1,5 @@
 fn main() -> Result<(), std::io::Error> {
+    std::env::set_var("PROTOC", protobuf_src::protoc());
```

Comment:
should we only do this if `PROTOC` is unset to honor the caller's environment?

Reply:
Ah, yep, totally should